client/app/dbaas/metrics/header/metrics-header.controller.js

```diff
@@ -30,6 +30,10 @@
                 title: this.$translate.instant("metrics_guides_protocoles"),
                 list: this.getProtocolDocs()
             });
```

Comment:
missing line break

Reply:
Same as previous comment. Line break there but not displayed by github.

Reply:
Done

Reply:
Done